Libraries/Lists/VirtualizedList.js

```diff
@@ -83,6 +83,21 @@ type OptionalProps = {
   initialScrollIndex?: ?number,
   keyExtractor: (item: Item, index: number) => string,
   /**
+   * Rendered when the list is empty. Can be a React Component Class, a render function, or
+   * a rendered element.
+   */
+  ListEmptyComponent?: ?(ReactClass<any> | React.Element<any>),
+  /**
+   * Rendered at the bottom of all the items. Can be a React Component Class, a render function, or
+   * a rendered element.
+   */
+  ListFooterComponent?: ?(ReactClass<any> | React.Element<any>),
```

Comment:
I think ListFooterComponent doesn't accept React.Element, just ReactClass. 
I got this when I tried ListFooterComponent={<OneComponent/>}

![](https://www.dropbox.com/s/39691uy6zdorx9a/Captura%20de%20pantalla%202017-05-14%2011.57.02.png?dl=1)